src/screens/LoginPage/LoginPage.tsx

```diff
@@ -57,6 +64,53 @@ function loginPage(): JSX.Element {
   const [showPassword, setShowPassword] = useState<boolean>(false);
   const [showConfirmPassword, setShowConfirmPassword] =
     useState<boolean>(false);
+  const [showAlert, setShowAlert] = useState<PasswordValidation>({
+    lowercaseChar: true,
+    uppercaseChar: true,
+    numericValue: true,
+    specialChar: true,
+  });
+
+  const passwordValidationRegExp = {
+    lowercaseCharRegExp: new RegExp('[a-z]'),
+    uppercasaeCharRegExp: new RegExp('[A-Z]'),
```

Comment:
Fix the spelling for upperCaseCharRegExp

Reply:
@rishav-jha-mech 
Completed this via https://github.com/PalisadoesFoundation/talawa-admin/pull/1376/commits/7e1dea4c257ec1a903d4b9a6e04d0f48b76e490c